README.md

```diff
@@ -1,92 +1,35 @@
+## Contents
+
+- [Overview](#overview)
+- [Features](#features)
+- [Admin Guide](/docs/admin-guide.md)
+- Contribute
+  - [Development](#development)
+  - [Maintainers](#maintainers)
+  - [Platform and Tools](#platform-and-tools)
+  - [Set up CircleCI](#set-up-circleci)
+- [License](/LICENSE)
```

Comment:
I think we can remove the following sections from the document, as they are either outdated or contain irrelevant information:

- Maintainers
- Platform and Tools
- Set up CircleCI

In the "Development" section, we can just have this sentence:

```
Read the [Developer Workflow](https://developers.mattermost.com/extend/plugins/developer-workflow/) and [Developer Setup](https://developers.mattermost.com/extend/plugins/developer-setup/) documentation for more information about developing and extending plugins.
```

Reply:
done.